subsys/net/ip/net_context.c

```diff
@@ -1638,6 +1638,7 @@ static int context_sendto(struct net_context *context,
 
```

Comment:
Just a tiny nit, the commit subject contains unnecessary information as
`net: context: log on net_pkt alloc fail`
would be enough. Not sure if this is worth changing, use your judgement here.

Reply:
done